block.go

```diff
@@ -179,11 +180,14 @@ type BlockMeta struct {
 
 // BlockStats contains stats about contents of a block.
 type BlockStats struct {
-	NumSamples    uint64 `json:"numSamples,omitempty"`
-	NumSeries     uint64 `json:"numSeries,omitempty"`
-	NumChunks     uint64 `json:"numChunks,omitempty"`
-	NumTombstones uint64 `json:"numTombstones,omitempty"`
-	NumBytes      int64  `json:"numBytes,omitempty"`
+	NumSamples        uint64 `json:"numSamples,omitempty"`
+	NumSeries         uint64 `json:"numSeries,omitempty"`
+	NumChunks         uint64 `json:"numChunks,omitempty"`
+	NumTombstones     uint64 `json:"numTombstones,omitempty"`
+	NumBytesChunks    int64  `json:"numBytesChunks"`
```

Comment:
They are lacking `omitempty` intentionally? To have known bytes even if the value is 0 right? Wonder if we really want to break consistency here.

Also also why we are breaking consistency in types? I would vote for `uint64` as we don't expect negative values, right?

Reply:
Other question. Can we remove all `NumBytes` things at all? Reasons in review summary.